include/cuco/types.cuh

```diff
@@ -58,6 +58,15 @@ CUCO_DEFINE_STRONG_TYPE(sketch_size_kb, double);
  */
 CUCO_DEFINE_STRONG_TYPE(standard_deviation, double);
 
+/**
+ * @brief A strong type wrapper `cuco::precision` for specifying the HyperLogLog precision
+ * parameter of `cuco::hyperloglog(_ref)`.
+ *
+ * @note Precision `p` determines the number of registers as `2^p`. Valid range is typically [4,
+ * 18].
+ */
```

Comment:
`precision` is a quite generic term. I can't come up with something better but reserving this term exclusively for HLL feels not right.